### PR TITLE
[author] Add pretty-checkbox v3.0.0, and update package.json

### PR DESCRIPTION
**Pull request by lokesh-coder**
Pull request for issue: #12037

Updated newer version and base path.

Question: Will this change affect existing CDN links of older versions?

Thanks!

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@lokesh-coder it won't change affect existing CDN links of older versions :) Good question!

File: ajax/libs/pretty-checkbox/package.json [hunk 1]
```diff
@@ -20,9 +20,9 @@
   "npmName": "pretty-checkbox",
   "npmFileMap": [
     {
-      "basePath": "src",
+      "basePath": "dist",
       "files": [
-        "**/pretty*.+(css|js|map)"
+        "**/pretty-checkbox*.+(css|js|map)"
```
Comment (sashberd) [inline ajax/libs/pretty-checkbox/package.json]: For dist folder we should use "**/*" pattern

Comment (lokesh-coder) [reply ajax/libs/pretty-checkbox/package.json]: Oh okay. Just updated the file.

Comment (lokesh-coder) [reply ajax/libs/pretty-checkbox/package.json]: Hi, I am getting this error in CI. "filename from package.json exists and is a file (not a directory)". Can you please help me why this error is thrown?

image

Comment (sashberd) [reply ajax/libs/pretty-checkbox/package.json]: @lokesh-coder Please remove version from package.json or add verson 3.0.0 folder with its assets to you PR

**Comment by PeterDaveHello** [comment]
Thanks @lokesh-coder

**Pull request merged**